webapp/platform/components/src/legacy_modal/index.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState, ReactElement} from 'react'
+import { Modal, ModalProps } from 'react-bootstrap'
+
+import { FocusTrap } from '../focus_trap'
+
+/**
+ * This is a wrapper around the react-bootstrap Modal component that adds
+ * focus trap functionality. It is intended to be used in place of the
+ * react-bootstrap Modal component directly.
+ * Do not use this component for new modals. Use the GenericModal component instead
+ */
```

Comment:
Will updating the modals that come from the LHS to `GenericModal` work so we don't need to introduce this `LegacyModal`?